utils.js

```diff
@@ -32,6 +33,16 @@ function setPostFields(post) {
   post.attributes.published_on = date;
   post.attributes.readingTime = getReadingTime(post.attributes.content);
   post.attributes.image_url = post.attributes.image.data.attributes.url;
+  post.attributes.authorName =
```

Comment:
Let's make common author like,
`const author = post.attributes.author_id.data.attributes` then use it like, 
`post.attributes.authorName = author.username`